Typescript Issue - Type expected issue on line 20

https://github.com/iamkun/dayjs/blob/bc53b53561f223b5efe7c4fbb76587f083ec8230/types/index.d.ts#L20.

node_modules/dayjs/index.d.ts:20:29 - error TS1110: Type expected.

tsc Version 3.9.5

issue from commit https://github.com/iamkun/dayjs/pull/1289/commits/676d09b46aed143e2780c45f58cb1443ef6e2e33#diff-093ad82a25aee498b11febf1cdcb6546e4d223ffcb49ed69cc275ac27ce0ccceR20

Same issue as the initial report.

- node@14.13.0
- typescript@3.9.6
- dayjs@1.10.0

tsconfig.json:

```json
{
  "compilerOptions": {
    "sourceMap": false,
    "inlineSources": false,
    "inlineSourceMap": false,
    "noEmit": false,
    "declaration": true,
    "stripInternal": true,
    "outDir": "dist",
    "downlevelIteration": true,
    "esModuleInterop": true,
    "importHelpers": false,
    "module": "commonjs",
    "moduleResolution": "node",
    "noFallthroughCasesInSwitch": true,
    "removeComments": true,
    "target": "es6"
  },
  "include": [
    "src",
    "test"
  ]
}
```

test.ts:

```typescript
import dayjs, { Dayjs } from "dayjs";
```

build:

```bash
$ tsc -p tsconfig.json 
node_modules/dayjs/index.d.ts:20:29 - error TS1110: Type expected.

20   type UnitTypeLongPlural = `${UnitTypeLong}s`
                               ~~~

Found 1 error.
```

--- a/types/index.d.ts
+++ b/types/index.d.ts
@@ -17,7 +17,8 @@ declare namespace dayjs {
 
   type UnitTypeLong = 'millisecond' | 'second' | 'minute' | 'hour' | 'day' | 'month' | 'year' | 'date'
 
-  type UnitTypeLongPlural = `${UnitTypeLong}s`
+  type UnitTypeLongPlural = 'milliseconds' | 'seconds' | 'minutes' | 'hours' | 'days' | 'months' | 'years' | 'dates'
+  
   export type UnitType = UnitTypeLong | UnitTypeLongPlural | UnitTypeShort;
 
   export type OpUnitType = UnitType | "week" | "weeks" | 'w';

